### PR TITLE
feat(wallet): changing more labels from `wallet` to `account` wording

### PR DESCRIPTION
**Pull request by cmonti-bc**
## Description (optional)
More labels that were found with `wallet` wording that should change into `account` wording.

### CODE REVIEW

**Review by jtk-bc — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 1]
```diff
@@ -2914,4 +2906,4 @@ type MessagesType = {
 }
 
 export type MessageIdType = keyof MessagesType
-export type DefaultMessageType = MessagesType[MessageIdType]
+export type DefaultMessageType = MessagesType[MessageIdType]
```
Comment (gsas-bc) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:2909]: Missing new line?

**Review by gsas-bc — DISMISSED** [review]

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts (hunk 1 above)
Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:2909]: Great catch, fixed.

**Review by schnogz — CHANGES_REQUESTED** [review]
I think these changes are too broad.  We still need to use the term Wallet in many of these contexts since they are using the Wallet product.   @amadeo-bc thoughts?

**Comment by amadeo-bc** [comment]
Reopening. I need to look at this before we release unified signup.

**Pull request closed without merging**